environments/development/common/rds.tf

```diff
@@ -25,6 +26,35 @@ module "postgres" {
   ]
 }
 
+# Admin Postgres
+module "postgres_admin" {
+  source = "../../common/rds"
+
+  environment    = var.environment
+  name           = "PostgresAdmin"
+  engine         = "postgres"
+  engine_version = "13.11"
```

Comment:
Just question but is this the version we are currently using?

Reply:
As far as I know. @willfish said it was fine 😇

Reply:
realise its just a small minor difference though